package.json

```diff
@@ -36,6 +41,8 @@
     "@types/react": "^16.9.35",
     "babel-jest": "^26.0.1",
     "babel-plugin-graphql-tag": "^2.5.0",
+    "cypress": "^4.8.0",
+    "dotenv": "^8.2.0",
```

Comment:
@lauragift21 do you need `dotenv` or is it leftover?

Reply:
I don't think so I'll take it out.